.buildkite/pipeline.yml

```diff
@@ -309,7 +309,7 @@ steps:
     timeout_in_minutes: 30
     depends_on: 'windows-2020-fixture'
     agents:
-      queue: windows-general-wsl
+      queue: windows-unity-wsl
```

Comment:
Actually this can stay on the general queue because it doesn't need a Unity license.